tools/mgbalenacli/tools.go

```diff
@@ -0,0 +1,86 @@
+package mgbalenacli
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/magefile/mage/mg"
+	"go.einride.tech/mage-tools/mgpath"
+	"go.einride.tech/mage-tools/mgtool"
+)
+
+const (
+	toolName   = "balena-cli"
+	binaryName = "balena"
+	version    = "v13.1.11"
+)
+
+func Command(ctx context.Context, args ...string) *exec.Cmd {
+	mg.CtxDeps(ctx, Prepare.Balena)
+	return mgtool.Command(ctx, mgpath.FromBins(binaryName), args...)
+}
+
+func Whoami(ctx context.Context) (WhoamiInfo, error) {
+	cmd := Command(ctx, "whoami")
+	cmd.Stdout = nil
+	output, err := cmd.Output()
```

Comment:
Shouldnt this use the new `Output` function? :)

Reply:
Nope, this function needs to return an error so the caller can intercept failure (i.e. when not logged in for example)

Reply:
:+1: